examples/nlp/language_modeling/conf/megatron_bert_config.yaml

```diff
@@ -56,15 +56,19 @@ model:
   hidden_size: 768
   ffn_hidden_size: 3072 # Transformer FFN hidden size. Usually 4 * hidden_size.
   num_attention_heads: 12
+  skip_head: False
+  transformer_block_type: post_ln
```

Comment:
Default is preln?

Reply:
previous default is preln. you want me to change it back? I am okay with it

Reply:
@ericharper  What do you think ?